ompi/mpi/fortran/mpif-h/comm_spawn_f.c

```diff
@@ -119,9 +118,7 @@ void ompi_comm_spawn_f(char *command, char *argv, MPI_Fint *maxprocs,
         opal_argv_free(c_argv);
     }
     if (!OMPI_IS_FORTRAN_ERRCODES_IGNORE(array_of_errcodes)) {
-	OMPI_ARRAY_INT_2_FINT(array_of_errcodes, size);
-    } else {
-	OMPI_ARRAY_FINT_2_INT_CLEANUP(array_of_errcodes);
```

Comment:
I think you still need the CLEANUP here, right?  I.e., for the non-IGNORE case, we have to have a matching CLEANUP for the ALLOC on new line 94, right?

Reply:
`OMPI_ARRAY_INT_2_FINT()` includes the cleanup of the array(will double check tomorrow)

Reply:
Ah yes, you are correct.  That's confusing.  ☹️  But so be it!